test/unit/dredd-command-test.coffee

```diff
@@ -272,6 +272,61 @@ describe "DreddCommand class", () ->
       it 'should save configuration', ->
         assert.isTrue configUtilsStub.save.called
 
+    describe '"init"', ->
```

Comment:
Yes, but helping with coverage for lines with others options (php, python, perl).